server/basic-auth.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const env = process.env;
+
+module.exports = {
+  //TODO: research loopback built-in mechanism for per-resource basic auth
```

Comment:
simple basic auth module. maybe there's a builtin option within loopback? or we decide to use token auth...